Source/TrackableEntities.Client/TrackableExtensions.cs

```diff
@@ -60,7 +79,7 @@ internal static class TrackableExtensions
         /// <param name="visitationHelper">Circular reference checking helper</param>
         /// <param name="isManyToManyItem">True is an item is treated as part of an M-M collection</param>
         public static void SetState(this ITrackable item, TrackingState state, ObjectVisitationHelper visitationHelper,
-            bool isManyToManyItem = false)
+                bool isManyToManyItem = false)
         {
             ObjectVisitationHelper.EnsureCreated(ref visitationHelper);
 
```

Comment:
Ignore my previous comment here .. looks like the only change was in the spacing.  Can you please discard the spacing changes? :-)